packages/welldone-wallet/src/lib/welldone.ts

```diff
@@ -329,6 +329,29 @@ const WelldoneWallet: WalletBehaviourFactory<InjectedWallet> = async ({
 
       return results;
     },
+
+    buildImportAccountsUrl() {
+      return `https://docs.welldonestudio.io/contribution/202211-batch-import`;
+    },
+
+    async importAccountsInSecureContext({ accounts }) {
```

Comment:
Let's remove `importAccountsInSecureContext` on this PR if it is not ready from WELLDONE injected wallet side.